src/mmw/js/src/analyze/models.js

```diff
@@ -472,15 +492,48 @@ function createAnalyzeTaskGroupCollection(aoi, wkaoi) {
             .value();
     }
 
+    // Remove PA only tasks first, if needed,
+    // as it is always a lazy-load option.
+    if (!utils.isInPAStrict(aoi)) {
+        var isPATask = function(task) {
+            return task.name.startsWith('pa_');
+        };
+
+        taskGroups = _(taskGroups)
+            // Remove tasks that are PA only
+            .map(function(tg) {
+                tg.tasks = _.reject(tg.tasks, isPATask);
+                return tg;
+            })
+            // Remove task groups with no tasks
+            .filter(function(tg) {
+                return !_.isEmpty(tg.tasks);
+            })
+            .value();
+    }
+
     if (!utils.isInDrb(aoi)) {
         var isDrbTask = function(task) {
             return task.name.startsWith('drb');
```

Comment:
Let's do this, and rename the tasks to be `pointsource_conus`, `pointsource_pa`, and `pointsource_drb`, to follow the pattern of `streams_nhd`, etc.

```suggestion
            return task.name.startsWith('drb') || task.name.endsWith('_drb');
```

Reply:
Updated in e5ec7dcf235727f76c53060e2ebf69a7c3f57281